content/en/dashboards/ddsql_editor/reference/scalar_functions.md

```diff
@@ -0,0 +1,246 @@
+
+### string_to_array
+| Name | Return type | Description |
+|------|-------------|-------------|
+| string_to_array(text *s*, text *delimiter*, [text *nullString1*, *nullString2* ...]) | array | Returns an array of substrings obtained by splitting the input string *s*, using the specified *delimiter*. The third argument is optional and specifies substrings that are replaced with `NULL`. |
+
+### array_to_string
+| Name | Return type | Description |
+|------|-------------|-------------|
+| array_to_string(array *a*, text *delimiter*, [text *nullString1*, *nullString2* ...]) | string | Concatenates array elements using the supplied *delimiter* and optional null strings (substrings to replace with `NULL`). |
+
+### unnest
+| Name | Return type | Description |
+|------|-------------|-------------|
+| unnest(array *a*) | variable | Returns each element in the array as a separate row. The return type is the element type of the array. `unnest` can only be used in the `SELECT` clause of a query. If other columns are selected with `unnest`, the value at each row in the table is repeated at each "output" row with each unnested element. If multiple columns are being unnested, all the unnested columns are "zipped up" together, with `NULL` filling in the "output" values for shorter arrays. |
+
+## Date/time functions and operators
+
+### date_trunc
+| Name | Return type | Description |
+|------|-------------|-------------|
+| date_trunc(string *precision*, timestamp *t*) | timestamp | Truncates the timestamp to the chosen *precision* ("second", "minute", "hour", "day", "week", "month", or "year"). |
+
+### date_diff
+| Name | Return type | Description |
+|------|-------------|-------------|
+| date_diff(string *precision*, timestamp *t1*, timestamp *t2*) | integer | Returns the difference between two dates, in the precision specified. *precision* is one of("second", "minute", "hour", "day", "week", "month", "quarter", "year") |
+
+<!-- QUERY below function description is unclear. Time in seconds since the Unix epoch? -->
+
+### to_timestamp
+| Name | Return type | Description |
+|------|-------------|-------------|
+| to_timestamp(numeric *n*) | timestamp | Transforms *n* into a timestamp, considering *n* as the time in seconds. |
+
+### Conditional expressions
+
+### coalesce
+| Name | Return type | Description |
+|------|-------------|-------------|
+| coalesce(expr *x*, *y*, ...) | variable | Returns the first non-null expression. |
+
+### nullif
+| Name | Return type | Description |
+|------|-------------|-------------|
+| nullif(expr *x*, expr *y*) | variable | Returns `NULL` if both arguments are equal. Otherwise, returns *x*. |
+
+## JSON functions and operators
+
+<!-- QUERY: An example of the below function would be helpful, if we're going to include the function in the docs. -->
+
+### json_extract_path_text
```

Comment:
Are the parameters on this one represented correctly, or are there possibly multiple path arguments?

Reply:
this test file might be of help [x](https://github.com/DataDog/dd-go/blob/e7846544686a3f8de508e1327075f3631c877179/apps/beagle/internal/sqllogictest/testdata/json_functions.test#L24-L60). the second argument can be a succession of path_elems, see also the doc [here](https://www.postgresql.org/docs/12/functions-json.html#FUNCTIONS-JSON-PROCESSING-TABLE) that's clearer